impeller/aiks/save_layer_utils.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_IMPELLER_AIKS_SAVE_LAYER_UTILS_H_
+#define FLUTTER_IMPELLER_AIKS_SAVE_LAYER_UTILS_H_
+
+#include <memory>
+#include <optional>
+
+#include "impeller/aiks/image_filter.h"
+#include "impeller/aiks/paint.h"
+
+namespace impeller {
+
+/// Compute the coverage of the subpass in the subpass coordinate space.
+///
+/// A return value of std::nullopt indicates that the coverage is empty or
+/// otherwise does not intersect with the parent coverage limit and should
+/// be discarded.
+std::optional<Rect> ComputeSaveLayerCoverage(
+    Rect content_coverage,
+    const std::shared_ptr<ImageFilter>& backdrop_filter,
+    const Paint& paint,
+    const Matrix& effect_transform,
+    const Rect& coverage_limit,
+    bool user_provided_bounds = false);
```

Comment:
Looks like this is dead code. Did you forget to push the entity_pass.cc changes?

Reply:
Oh, I didn't change entity pass itself, just staged this separately. I could do that too though, if you'd like

Reply:
Please do. It would be helpful to see user_provided_bounds being used in context.